nncf/quantization/quantize.py

```diff
@@ -57,8 +58,23 @@ def quantize(model: ModelType,
     """
     backend = get_backend(model)
     if backend == BackendType.OPENVINO:
-        from nncf.openvino.quantization.helpers import quantize_impl
+        from nncf.openvino.quantization.quantize import quantize_impl
         return quantize_impl(model, calibration_dataset, preset, target_device, subset_size,
-                             fast_error_correction, model_type, ignored_scope)
+                             fast_bias_correction, model_type, ignored_scope)
+
+    if backend == BackendType.ONNX:
+        from nncf.onnx.quantization.quantize import quantize_impl
+        return quantize_impl(model, calibration_dataset, preset, target_device, subset_size,
+                             fast_bias_correction, model_type)
+
+    if backend == BackendType.TENSORFLOW:
+        from nncf.tensorflow.quantization.quantize import quantize_impl
+        return quantize_impl(model, calibration_dataset, preset, target_device, subset_size,
+                             fast_bias_correction, model_type)
+
+    if backend == BackendType.TORCH:
+        from nncf.torch.quantization.quantize import quantize_impl
+        return quantize_impl(model, calibration_dataset, preset, target_device, subset_size,
+                             fast_bias_correction, model_type)
```

Comment:
Should the `ignored_scope` parameter be supported here?

Reply:
Thanks for the comment. Done.